README.md

```diff
@@ -173,8 +173,6 @@ Code template was mainly taken from [@Ludeeus](https://github.com/ludeeus)'s [in
 
 Under the hood the integration uses [glocaltokens](https://github.com/leikoilja/glocaltokens) python package.
 
----
-
 [buymecoffee]: https://www.buymeacoffee.com/leikoilja
 [buymecoffeebadge]: https://img.shields.io/badge/buy%20me%20a%20coffee-donate-yellow.svg?style=for-the-badge
```

Comment:
Let's also clean the badges we are not using, like these :)

Reply:
Don't you know to get some coffee? 😀

Reply:
hahha i sure do 😄 
 but c'mon those dont work, there are not that many people willing to contribute to open source in $  😅

Reply:
Plus it's a team work, how are we gonna split all the bucks? 😃

Reply:
I was recently surprised how few people are sponsoring HA authors via github sponsors. 🙁

Reply:
That does not surprise me unfortunately 😞 I often read some of the issues in home-assistant/core or on the forum and a lot of people often forget that people do the work for free (excluding Nabu Casa).

Reply:
yeeeeh, unfortunatelly that's the world we are living in 😄  
we can ofc think about starting to monetarizing the integration once we have a solid base of users 😅  jk

Reply:
Like a 1 dollar per device? 🤣